_dropin-templates/dropin-installation.mdx

```diff
@@ -0,0 +1,248 @@
+  order: 2
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+
+The DROPIN_NAME drop-in component, like our other drop-in components, are designed for the browser's JavaScript runtime without the need for a bundler. You can also install and execute drop-in components in a build-time environment with bundlers like Vite. The installation steps for both runtime and build-time environments are the same after the initial drop-in component package imports.
+
+## Step-by-step
+
+Use the following steps to install the DROPIN_NAME component:
+
+:::note[Edge Delivery Service]
+These steps are specific for EDS projects, and may not necessarily be the same for other frameworks such as a static HTML or React framework.
+:::
+
+<Tasks>
+
+<Task>
+
+### Install the packages
+
+Use a CDN or NPM (recommended for performance) to install the drop-in component tools (`@dropins/tools`) and DROPIN_NAME (`@dropins/storefront-DROPIN_PACKAGE`) packages.
+
+<Tabs>
+  <TabItem label="NPM" icon="seti:npm">
+
+    ```bash frame="none"
+    npm install @dropins/tools @dropins/storefront-DROPIN_PACKAGE
+    ```
+
+  </TabItem>
+  <TabItem label="CDN" icon="external">
+
+    ```html title="index.html" del={"Replace with actual URLs":5-7}
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Your Storefront</title>
+
+      <script src="https://cdn.jsdelivr.net/npm/@dropins/tools@latest"></script>
+      <script src="https://cdn.jsdelivr.net/npm/@dropins/storefront-DROPIN_PACKAGE@latest"></script>
+    </head>
+    ```
+
+  </TabItem>
+</Tabs>
+
+:::note[Install @dropins/tools]
+All drop-in components require the `@dropins/tools` package. This package contains the libraries that drop-in components need to initialize and communicate, including `fetch-graphql`, `event-bus`, and `initializer` utilities.
+:::
+
+</Task>
+
+<Task>
+### Map the packages
+
+In the `<head>` tag of your `index.html` or `head.html` file, use an `importmap` pointed to the `node_modules` directory, a **custom local directory**, or **CDN** (for run-time environments).
+
+<Tabs>
+<TabItem label="node_modules" icon="seti:npm">
+    This example shows an `importmap` added to a `head.html` The `importmap` points both packages to the local `node_modules` directory that contains your installed drop-in component files from the drop-in component tools (@dropins/tools) and the DROPIN_NAME (@dropins/storefront-DROPIN_PACKAGE):
+
+  ```html title="head.html"
+  <script type="importmap">
+    {
+      "imports": {
+        "@dropins/tools/": "/node_modules/@dropins/tools/",
+        "@dropins/storefront-DROPIN_PACKAGE/": "/node_modules/@dropins/storefront-DROPIN_PACKAGE/",
```

Comment:
`node_modules` folders are not published in Edge Delivery Services. I’m concerned this may confuse those who are not paying close attention to the details. Should we instead use the paths currently defined in our Commerce Boilerplate?  https://github.com/hlxsites/aem-boilerplate-commerce/blob/60a337926133ee00694714698aabb65f44ff0d96/head.html#L17

Reply:
@fnhipster I think I simplified this area correctly, but please double check.